kernel/realsense/d4xx.c

```diff
@@ -5530,10 +5544,15 @@ static int ds5_probe(struct i2c_client *c, const struct i2c_device_id *id)
 
 static int ds5_remove(struct i2c_client *c)
 {
+#ifdef CONFIG_VIDEO_D4XX_SERDES
```

Comment:
Why do we want it here and not after line 5555 where they use it?

Reply:
[ISO C90 forbids mixed declarations and code in C](https://www.kernel.org/doc/html/v5.10/process/programming-language.html)